client/src/components/WelcomeMessage/WelcomeMessage.jsx

```diff
@@ -0,0 +1,14 @@
+function WelcomeMessage() {
+  return (
+    <section className="welcomeArea">
+      <h2>Welcome to Good PR</h2>
+      <p className="welcome-msg">
+        A tracker that helps you visualize the contribution of team members at
```

Comment:
Suggestion? I would add something like this after welcome to good pr bit: 
A Code Your Future Final Projects Pull Request (PR) Tracker app,  which allows you to easily visualise each team member’s contributions to their final project, by accurately monitoring the number of Pull Requests they have made.